packages/salesforcedx-vscode-core/src/commands/forceCreateManifest.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { ComponentSet } from '@salesforce/source-deploy-retrieve';
+import * as fs from 'fs';
+import { join } from 'path';
+import { format } from 'util';
+import * as vscode from 'vscode';
+import { nls } from '../messages';
+import { getRootWorkspacePath } from '../util';
```

Comment:
do we have a preferred import order?

Reply:
It's to be sorted by alphabet (of the name of the module being imported).  Our linter spits out an error if they are alphabetized.